mm/kfence/core.c

```diff
@@ -548,6 +548,14 @@ static void toggle_allocation_gate(struct work_struct *work)
 	if (!READ_ONCE(kfence_enabled))
 		return;
 
+	/*
```

Comment:
I would have probably put this comment before the function, but don't have a strong preference.

Reply:
Done.